app/src/shared/utils/sessionToken.test.js

```diff
@@ -32,11 +25,18 @@ describe('session token utility', () => {
 
         it('gives the stored value before its expiration', () => {
             global.localStorage.setItem(SESSION_TOKEN_KEY, 'token')
-            global.localStorage.setItem(SESSION_LOGIN_TIME, new Date())
+            global.localStorage.setItem(SESSION_LOGIN_TIME, '2020-03-26T14:00:00.000Z')
+
+            jest.spyOn(Date, 'now').mockImplementationOnce(() => new Date('2020-03-26T15:00:00.000Z').getTime())
+
             expect(getToken()).toBe('token')
-            clock.tick(((EXPIRES_AT_VALID_HOURS * 3600) - 60) * 1000) // 60s before expiration
+
+            // 60s before expiration
+            jest.spyOn(Date, 'now').mockImplementationOnce(() => new Date('2020-03-26T19:59:00.000Z').getTime())
```

Comment:
Ok. So that's how you fake the clock now. Good to know. Seems more consistent with the rest.

Reply:
You can also fake timers with `jest.useFakeTimers()` and advance with eg. `jest.advanceTimersByTime(msToRun)`